### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

**Comment by kovacshuni** [comment]
+1

**Comment by tosan88** [comment]
+1

**Comment by Nesze** [comment]
+1

**Comment by kovacshuni** [comment]
+1

File: src/main/java/com/ft/message/consumer/config/MessageQueueConsumerConfiguration.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
     private final String groupName;
     private final String queueProxyHost;
     private final String queue;
-    private int backoffPeriod;
+    private final int backoffPeriod;
```
Comment (tamas-molnar) [inline src/main/java/com/ft/message/consumer/config/MessageQueueConsumerConfiguration.java]: What does the backoffPeriod represent and what is it's time unit?

File: src/main/java/com/ft/message/consumer/proxy/MessageQueueProxyServiceImpl.java [hunk 2]
```diff
@@ -33,7 +33,7 @@ public URI createConsumerInstance() {
             clientResponse = proxyClient.resource(uri)
                     .header("Content-Type", "application/json")
                     .header("Host", configuration.getQueue())
-                    .post(ClientResponse.class);
+                    .post(ClientResponse.class, "{\"auto.offset.reset\": \"smallest\", \"auto.commit.enable\": \"true\"}");
             if (clientResponse.getStatus() != 200) {
```
Comment (tamas-molnar) [inline src/main/java/com/ft/message/consumer/proxy/MessageQueueProxyServiceImpl.java]: Consider using constants for HTTP return codes.

**Pull request merged**